civis/utils/_jobs.py

```diff
@@ -26,7 +30,41 @@ def run_job(job_id, api_key=None, client=None):
     if client is None:
         client = APIClient(api_key=api_key, resources='all')
     run = client.jobs.post_runs(job_id)
-    return CivisFuture(client.jobs.get_runs,
-                       (job_id, run['id']),
-                       client=client,
-                       poll_on_creation=False)
+    fut = CivisFuture(client.jobs.get_runs,
+                      (job_id, run['id']),
+                      client=client,
+                      poll_on_creation=False)
+    if wait:
+        fut.result()
+    return fut
```

Comment:
The user will still have to call `.result()` to get the contents, right? Maybe a better solution is to add a method to `CivisFuture` like `.block()` or `.wait()` and recommend that users call this function like
```python
run_job(id, ...).block()
```
to force waiting for completion.